tutorbackup/templates/backup/build/backup/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/ubuntu:20.04
+FROM docker.io/ubuntu:22.04
```

Comment:
`CHANGELOG.md` should mention that this change requires an image rebuild.

Reply:
Ack, done.